index.js

```diff
@@ -118,12 +157,22 @@ const init = (HTTPServer, serverConfig) => {
   config.userActiveTimeout = config.dynamic && serverConfig && !isNaN(serverConfig.userActiveTimeout) ? serverConfig.userActiveTimeout : USER_ACTIVE_TIMEOUT_DEFAULT
   config.limit = config.dynamic && serverConfig && !isNaN(serverConfig.requestLimit) ? serverConfig.requestLimit : LIMIT_DEFAULT
   config.interval = config.dynamic && serverConfig && !isNaN(serverConfig.requestInterval) ? serverConfig.requestInterval : INTERVAL_DEFAULT
+  config.userTableMaxSize = serverConfig && !isNaN(serverConfig.userTableMaxSize) ? serverConfig.userTableMaxSize : USER_TABLE_MAX_SIZE_DEFAULT
   config.logging = serverConfig && serverConfig.logging ? serverConfig.logging : LOGGING_DEFAULT
   config.headerTimeout = serverConfig && serverConfig.headerTimeout ? serverConfig.headerTimeout : HEADER_TIMEOUT_DEFAULT
+  const userTableClearTimeout = serverConfig && !isNaN(serverConfig.userTableClearTimeout) ? serverConfig.userTableClearTimeout : USER_TABLE_CLEAR_TIMEOUT
   
   if (config.sl || config.all) {
     slowloris(HTTPServer, config.headerTimeout)
   }
+  if(config.rl && !_clearIPTableInterval) {
+    _clearIPTableInterval = setInterval(() => {
```

Comment:
Is the reason for assigning `setInterval()` to a global variable that it shouldn't be reassigned if `init()` is called multiple times?

Reply:
I believe that the global variable can be removed, since a user isn't "supposed" to init dostroy several times. If that is a risk maybe we could have a check that says "dostroy has already been initialized".

Reply:
Yes @josefinnilsson

Reply:
Remove as per @helenaalinder 's suggestion to impose this limit on the whole init func.

Reply:
Actually init() limit removed alltogether